src/core/ext/filters/client_channel/resolver/dns/c_ares/dns_resolver_ares.cc

```diff
@@ -353,12 +366,14 @@ void AresDnsResolver::OnResolvedLocked(grpc_error_handle error) {
   } else {
     GRPC_CARES_TRACE_LOG("resolver:%p dns resolution failed: %s", this,
                          grpc_error_std_string(error).c_str());
-    std::string error_message =
-        absl::StrCat("DNS resolution failed for service: ", name_to_resolve_);
-    result_handler_->ReturnError(grpc_error_set_int(
-        GRPC_ERROR_CREATE_REFERENCING_FROM_COPIED_STRING(error_message.c_str(),
-                                                         &error, 1),
-        GRPC_ERROR_INT_GRPC_STATUS, GRPC_STATUS_UNAVAILABLE));
+    std::string error_message;
+    grpc_error_get_str(error, GRPC_ERROR_STR_DESCRIPTION, &error_message);
```

Comment:
Since we're no longer seeing `GRPC_ERROR_INT_GRPC_STATUS` here, I want to make sure we're still filling this into RPC error message details.

Can we patch in the test for this in https://github.com/grpc/grpc/pull/28252? (or we can merge that separately). I've tested that this works on current master branch.

Reply:
Sure, sounds good.  I've approved that PR, so let's get that merged, and then I'll make sure it passes with this code.  (I think it will.)

Reply:
I've merged in your change, and the test does pass.  PTAL.